packages/strapi-plugin-upload/admin/src/components/InputMedia/index.js

```diff
@@ -182,6 +186,8 @@ InputMedia.propTypes = {
   onChange: PropTypes.func.isRequired,
   type: PropTypes.string.isRequired,
   value: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
+  id: PropTypes.string.isRequired,
```

Comment:
The `id` and `error` should be `PropTypes.string` 

You will need to add them to defaultProps:

```
InputMedia.propTypes = {
  attribute: PropTypes.shape({
    allowedTypes: PropTypes.arrayOf(PropTypes.string),
    multiple: PropTypes.bool,
    required: PropTypes.bool,
    type: PropTypes.string,
  }).isRequired,
  error: PropTypes.string,
  id: PropTypes.string,
  label: PropTypes.string,
  name: PropTypes.string.isRequired,
  onChange: PropTypes.func.isRequired,
  type: PropTypes.string.isRequired,
  value: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
};

InputMedia.defaultProps = {
  id: null,
  error: null,
  label: '',
  value: null,
};
```

Reply:
Updated